desktop/cmp/panel/impl/PanelHeader.ts

```diff
@@ -41,57 +62,57 @@ export const panelHeader = hoistCmp.factory({
             sideCls = `xh-panel-header--${side}`,
             compactCls = compact ? 'xh-panel-header--compact' : null;
 
-        // 1) Classic "top" title bar
-        if (!collapsed || vertical || isModal) {
-            return hbox({
-                className: classNames(className, compactCls),
+        // Return null if nothing to display.
+        if (isNil(displayedTitle) && isNil(displayedIcon) && isEmpty(displayedHeaderItems)) {
+            return null;
+        }
+
+        // Return a vertically oriented header if collapsed to left or right side.
+        if (isCollapsedToLeftOrRight) {
+            return vbox({
+                className: classNames(className, sideCls, compactCls),
+                flex: 1,
                 items: [
-                    icon || null,
+                    displayedIcon,
                     displayedTitle
                         ? box({
                               className: titleCls,
                               flex: 1,
                               item: span({className: `${titleCls}__inner`, item: displayedTitle})
                           })
                         : filler(),
-                    hbox({
-                        className: 'xh-panel-header__items',
-                        items: [
-                            ...(!collapsed || isModal ? headerItems : []),
-                            modalButton({panelModel}),
-                            collapseButton({panelModel})
-                        ],
-                        onDoubleClick: e => e.stopPropagation()
-                    })
+                    ...displayedHeaderItems
                 ],
                 onDoubleClick
             });
         }
 
-        // 2) ...otherwise its a narrow, sidebar
-        return vbox({
-            className: classNames(className, sideCls, compactCls),
-            flex: 1,
+        // Return a standard horizontal header otherwise.
+        // Panel is expanded, modal, and/or collapsed to the top or bottom side.
+        return hbox({
+            className: classNames(className, compactCls),
             items: [
-                collapseButton({panelModel}),
-                icon || null,
+                displayedIcon,
                 displayedTitle
                     ? box({
                           className: titleCls,
+                          flex: 1,
                           item: span({className: `${titleCls}__inner`, item: displayedTitle})
                       })
-                    : null
+                    : filler(),
+                hbox({
```

Comment:
Curious about the decision to move these to the bottom for L/R panel? At the top the collapse button is at the same Y position as when expanded, but that paradigm makes less sense now the modal button is involved.

Is it just a desire to not show up to 3 icons before the title? Or wanting the flow to remain Icon > Title > Space > Buttons? I don't feel too strongly either way.